config.yaml.defaults

```diff
@@ -605,3 +605,5 @@ analysis_services:
   analysis_folder: analysis_data
   sn_analysis_service:
     port: 6801
+
+comment_folder: comment_data
```

Comment:
This should be something like:
`persistentdata/comments`

Reply:
cf. https://github.com/skyportal/skyportal/pull/3505

Reply:
Hi Josh. Yes, I saw the PR! I'll change that soon.